src/amber/support/file_encryptor.cr

```diff
@@ -1,29 +1,26 @@
 require "./message_encryptor"
 
 module Amber::Support
-  module FileEncryptor
-    def self.encryption_key
-      ENV["AMBER_ENCRYPTION_KEY"]? || begin
-        if File.exists?(".encryption_key")
-          File.open(".encryption_key").gets_to_end.to_s
-        else
-          raise Exceptions::EncryptionKeyMissing.new
-        end
-      end
-    end
+  SECRET_KEY  = "AMBER_ENCRYPTION_KEY"
+  SECRET_FILE = "./.encryption_key"
 
-    def self.read(filename : String, encryption_key = self.encryption_key)
+  module FileEncryptor
+    def self.read(filename : String, encryption_key = encryption_key)
       encryptor = Amber::Support::MessageEncryptor.new(encryption_key)
       encryptor.verify_and_decrypt(File.open(filename).gets_to_end.to_slice)
     end
 
-    def self.write(filename : String, body : (String | Bytes), encryption_key = self.encryption_key)
```

Comment:
`encryption_key = self.encryption_key` looks a lot better because it doesn't look like it's assigning itself to itself.